src/test/regress/citus_tests/upgrade/pg_upgrade_test.py

```diff
@@ -64,12 +64,12 @@ def citus_finish_pg_upgrade(pg_path, node_ports):
         utils.psql(pg_path, port, "SELECT citus_finish_pg_upgrade();")
 
 
-def stop_all_databases(old_bindir, new_bindir, old_datadir, new_datadir, config):
+def stop_all_databases(old_bindir, new_bindir, old_datadir, new_datadir, config, no_output=False, parallel=True):
```

Comment:
Why is the default for `parallel` is `True`?

Reply:
good question, I did it to stay in line with `stop_databases`. But since this function is only used in `atexit` that doesn't make sense.